src/Microsoft.Extensions.Configuration.FileProviderExtensions/FileConfigurationProvider.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public abstract class FileConfigurationProvider : ConfigurationProvider
```

Comment:
So the current difference in the two package dependencies are:

FileExtensions depends on Abstractions only.
FileProviderExtensions depends on the Configuration implementation and Physical file providers

I would assume this was factored this way for a reason, so we should probably be careful collapsing these together